wiki/People/Users_with_unique_titles/id.md

```diff
@@ -89,45 +85,52 @@ Sebagian besar kontes mapping resmi menawarkan gelar **Elite Mapper** sebagai ha
 
-Kontes mapping Aspire menawarkan variasi khusus dari gelar Elite Mapper, gelar ini dinamakan **Elite Mapper: Aspirant** sebagai bentuk pencapaian.
+osu! menganugerahkan gelar **Elite Mapper: Aspirant** kepada para pemenang kontes-kontes mapping Aspire.
 
-- ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000)
+- ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000)¹
 - ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699)
 - ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438)
 - ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619)
 - ![][flag_FI] [ProfessionalBox](https://osu.ppy.sh/users/3250792)
 - ![][flag_TW] [SnowNiNo_](https://osu.ppy.sh/users/2506267)
-- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
+- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)²
 - ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231)
 
-![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) menerima gelar ini setelah sebelumnya memiliki gelar Elite Mapper, jadi tim osu! meningkatkan levelnya menjadi **Elite Mapper II: Aspirant**. ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini lebih memilih untuk menggunakan gelar **osu!catch Champion**.
+¹ ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) saat ini menyandang gelar **Elite Mapper II: Aspirant** atas keberhasilannya dalam menjuarai [Monthly Beatmapping Contest #1](https://osu.ppy.sh/home/news/2013-10-20-monthly-beatmapping-contest-1-results) dan [Monthly Beatmapping Contest: Aspire (2014)](https://osu.ppy.sh/home/news/2014-11-13-monthly-beatmapping-contest-aspire-results) secara bersamaan.
+
+² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.
 
-### Pemenang kontes osu!remix
+### Pemenang kontes-kontes osu!remix
 
-Kontes osu!remix menghadiahkan gelar **osu!mixer** sebagai hadiah utama dari kontes tersebut.
+osu! menganugerahkan gelar **osu!mixer** kepada para pemenang kontes-kontes osu!remix.
 
-- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)
+- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)¹
 - ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574)
 
-### Artis unggulan
+¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) saat ini menyandang gelar **osu!mixer II** atas keberhasilannya dalam menjuarai dua kontes osu!remix yang berbeda, yakni [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) dan [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest).
+
+### Featured Artist
 
-[Artis unggulan](/wiki/Featured_Artists) yang memiliki akun osu! memiliki gelar **Featured Artist** untuk menunjukkan keterlibatan mereka dalam game ini.
+osu! menganugerahkan gelar **Featured Artist** kepada para [Featured Artist](/wiki/Featured_Artists) yang memiliki akun osu! sebagai salah satu bentuk balas jasa atas lagu-lagu yang telah mereka sediakan.
```

Comment:
```suggestion
osu! memberikan gelar **Featured Artist** kepada para [Featured Artist](/wiki/Featured_Artists) yang memiliki akun osu! sebagai salah satu bentuk balas jasa atas lagu-lagu yang telah mereka sediakan untuk osu!.
```

Reply:
similar to http://github.com/ppy/osu-wiki/pull/5622#discussion_r642110326 i don't think making the word `osu!` to be redundant here is a good thing :(